Makefile

```diff
@@ -12,18 +12,21 @@ docs:
 	cd docsrc/ && make github
 
 test:
-	pytest tests/unit/
-	pytest tests/issues/
-	pytest --nbval tests/notebooks/
+	pytest -m "not sparktest" tests/unit/
+	pytest -m "not sparktest" tests/issues/
+	pytest -m "not sparktest" --nbval tests/notebooks/
 	flake8 . --select=E9,F63,F7,F82 --show-source --statistics
 	
 test_cov:
-	pytest --cov=. tests/unit/
-	pytest --cov=. --cov-append tests/issues/
-	pytest --cov=. --cov-append --nbval tests/notebooks/
+	pytest -m "not sparktest" --cov=. tests/unit/
+	pytest -m "not sparktest" --cov=. --cov-append tests/issues/
+	pytest -m "not sparktest" --cov=. --cov-append --nbval tests/notebooks/
 	pandas_profiling -h
 	make typing
 
+test-spark:
+	pytest -m sparktest tests/unit/
```

Comment:
Shouldn't this have the --cov=. flag?

Reply:
I kept all the unit tests separate (one set of spark, one set for pandas for all the matrix of parameters) and finally one full set of tests (pandas + spark) to measure coverage

Reply:
see if this is ok? [tests](https://github.com/chanedwin/pandas-profiling/blob/develop-clone/.github/workflows/ci_test.yml) | [output](https://github.com/pandas-profiling/pandas-profiling/actions/runs/510768032)

Reply:
In that approach one combination runs twice. As long as Github Action's runners are so generously provided this shouldn't be a problem, but there is room too optimize. Fine with me for now.